### PR TITLE
this->fileSystem-> to CoolFileSystem::

### PR DESCRIPTION
**Pull request by Kameeno**

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -53,4 +53,4 @@ board = ${common.board}
 framework = ${common.framework}
 platform = ${common.platform}
 lib_deps = ${common.lib_deps}
-build_flags = ${common.build_flags} -DCOOL_LEVEL=COOL_TRACE -DCOOL_FUNC_TRACE
```
Comment (trap98) [inline platformio.ini:56]: Why ?

Comment (Kameeno) [reply platformio.ini:56]: it's strange, on my IDE this line was not removed...

Comment (Kameeno) [reply platformio.ini:56]: also on the branch it's right!

Comment (Kameeno) [reply platformio.ini:56]: aaah was a line return removed, check the diff!

Comment (trap98) [reply platformio.ini:56]: But you add it, you need to checkout the file from master and add it a second time, After you need to force push it

Comment (trap98) [reply platformio.ini:56]: On the same commit

**Review by trap98 — APPROVED** [review]

**Pull request merged**